loop/middlewares.js

```diff
@@ -92,6 +93,10 @@ function logMetrics(req, res, next) {
         }
```

Comment:
At this point the idea was to use the ``participants.length`` in case ``roomParticipantsCount`` is not present.